docs/DeveloperGuide.md

```diff
@@ -17,7 +17,7 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 * [Appendix E : Product Survey](#appendix-e--product-survey)
 
 
-## 1. Setting up
+<h2 name="setting-up">1. Setting up</h2>
```

Comment:
Do you need a name attribute on these? Can you just leave them as is? The `##` in markdown will be rendered as an `h2` when displayed